### ISSUE TITLE
Reproduce experiment

### ISSUE THREAD

**Issue by Coolshanlan** [TuanTNG/TemporalMaxer#1]
I performed `scripts/multithumos/train.sh` and `test.sh` to reproduce the experiment, but it can't achieve the official performance
<html>
<body>

Method | 0.1 | 0.2 | 0.3 | 0.4 | 0.5 | 0.6 | 0.7 | 0.8 | 0.9 | Avg
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Official | 49.1 | 47.5 | 44.3 | 39.4 | 33.4 | 26.5 | 17.4 | 9.1 | 2.24 | 29.9
Reproduce | 41.04 | 39.72 | 37.10 | 33.58 | 28.37 | 22.12 | 15.07 | 7.84 | 1.96 | 25.20

</body>
</html>

above is my reproduce result, how should I adjust the ymal setting to achieve the official result

Thanks

**Comment by TuanTNG** [comment]
Hi @Coolshanlan,

Thank you for your interest in our work. 
Sorry for the late reply.

- First, do you install the same package as provided in the readme? 

- Secondly, as soon as the training begins, a configuration file is produced that summarizes the entire model. The file is named `ckpt/multithumos/config.txt`. Would you be able to compare this file with your own to identify any discrepancies?
- Lastly, what epoch does the model stop at?

Please check them first and I will continue to invest further.

{'dataset': {'crop_ratio': [0.9, 1.0],
             'default_fps': None,
             'downsample_rate': 1,
             'feat_folder': './data/thumos/i3d_features',
             'feat_stride': 4,
             'file_ext': '.npy',
             'file_prefix': None,
             'force_upsampling': False,
             'input_dim': 1024,
             'json_file': './data/multithumos/multithumos.json',
             'max_seq_len': 2304,
             'num_classes': 65,
             'num_frames': 16,
             'trunc_thresh': 0.5},
 'dataset_name': 'multithumos',
 'devices': ['cuda:0'],
 'init_rand_seed': 1234567891,
 'loader': {'batch_size': 2, 'num_workers': 4},
 'model': {'assigner': {'center_radius': 1.5,
                        'cls_weight': 1.0,
                        'confuse_weight': 0.0,
                        'iou_weight': 2,
                        'keep_percent': 1.0},
           'backbone_arch': [3, 5],
           'backbone_type': 'convPooler',
           'embd_dim': 512,
           'embd_kernel_size': 3,
           'embd_with_ln': True,
           'fpn_dim': 512,
           'fpn_norm': 'layernorm',
           'fpn_start_level': 0,
           'fpn_type': 'identity',
           'fpn_with_ln': True,
           'head_dim': 512,
           'head_kernel_size': 3,
           'head_num_layers': 3,
           'head_with_ln': True,
           'input_dim': 1024,
           'max_buffer_len_factor': 6.0,
           'max_seq_len': 2304,
           'num_classes': 65,
           'regression_range': [(0, 4),
                                (4, 8),
                                (8, 16),
                                (16, 32),
                                (32, 64),
                                (64, 10000)],
           'scale_factor': 2,
           'test_cfg': {'duration_thresh': 0.05,
                        'ext_score_file': None,
                        'iou_threshold': 0.1,
                        'max_seg_num': 300,
                        'min_score': 0.001,
                        'multiclass_nms': True,
                        'nms_method': 'soft',
                        'nms_sigma': 0.5,
                        'pre_nms_thresh': 0.001,
                        'pre_nms_topk': 3000,
                        'voting_thresh': 0.7},
           'train_cfg': {'center_sample': 'none',
                         'clip_grad_l2norm': 1.0,
                         'cls_prior_prob': 0.01,
                         'head_empty_cls': [],
                         'init_loss_norm': 100,
                         'label_smoothing': 0.0,
                         'loss_weight': 1.0}},
 'model_name': 'TemporalMaxerArchitecture',
 'opt': {'epochs': 60,
         'learning_rate': 0.0001,
         'momentum': 0.9,
         'schedule_gamma': 0.1,
         'schedule_steps': [],
         'schedule_type': 'cosine',
         'type': 'AdamW',
         'warmup': True,
         'warmup_epochs': 5,
         'weight_decay': 0.05},
 'output_folder': './ckpt',
 'test_cfg': {'duration_thresh': 0.05,
              'ext_score_file': None,
              'iou_threshold': 0.1,
              'max_seg_num': 300,
              'min_score': 0.001,
              'multiclass_nms': True,
              'nms_method': 'soft',
              'nms_sigma': 0.5,
              'pre_nms_thresh': 0.001,
              'pre_nms_topk': 3000,
              'voting_thresh': 0.7},
 'train_cfg': {'center_sample': 'none',
               'clip_grad_l2norm': 1.0,
               'cls_prior_prob': 0.01,
               'head_empty_cls': [],
               'init_loss_norm': 100,
               'label_smoothing': 0.0,
               'loss_weight': 1.0},
 'train_split': ['validation'],
 'val_split': ['test']}

**Comment by Coolshanlan** [comment]
Thank you very much for your patient response. Upon inspection, it was found that the` thumos.tar.gz` file was damaged, which resulted in errors in the extracted i3d_feature. After re-downloading it, everything returned to normal. Many thanks to you and your excellent work.
